src/containers/_RegistrationContainer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Registration } from '../components'
+import { useHistory } from 'react-router-dom'
+
+const RegistrationContainer = () => {
+
+	const history = useHistory()
+
+	const onContinue = () => {
+		history.push('/signup/regform')
+	}
+
+	return (
+		<Registration>
+
+			<Registration.Inner>
+				<Registration.Image src="/images/registration.png"/>
+
+				<Registration.PreTitle>
+					STEP
+					<Registration.Span>1</Registration.Span>
+					OF
+					<Registration.Span>3</Registration.Span>
+				</Registration.PreTitle>
+
+				<Registration.Title>Finish Setting up your account. </Registration.Title>
+				<Registration.SubTitle>
+					Netflix is personalized for you. Create a password to watch Netflix on any device at any time.
+				</Registration.SubTitle>
+				<Registration.Button
```

Comment:
do you prefer that format of Components: w/ text + eventHandler + other?
or in-line Component?

Reply:
Not sure what you mean,